### PR TITLE
fix: `OwnershipCard` could be unscrollable when using together with `react-grid-layout`

### PR DESCRIPTION
**Pull request by JounQin**
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
  image
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/unlucky-apricots-call.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-org': patch
+---
+
+fix: `OwnershipCard` could be unscrollable when using together with `react-grid-layout`
```
Comment (vinzscam) [inline .changeset/unlucky-apricots-call.md:5]:
```suggestion
Fixed an issue where the `OwnershipCard` component failed to scroll properly when used alongside `react-grid-layout`.
```

**Review by vinzscam — CHANGES_REQUESTED** [review]
Thank you 🙏
Just a tiny suggestion on the changeset

**Comment by JounQin** [comment]
@vinzscam Any luck to review again?

**Review by benjdlambert — APPROVED** [review]
Thanks

**Pull request merged**